cmake/modules/hunter_download.cmake

```diff
@@ -142,9 +142,11 @@ function(hunter_download)
   set(HUNTER_GLOBAL_SCRIPT_DIR "${HUNTER_SELF}/scripts")
   set(HUNTER_PACKAGE_SCRIPT_DIR "${HUNTER_PACKAGE_SETUP_DIR}/scripts/")
   set(HUNTER_PACKAGE_HOME_DIR "${HUNTER_TOOLCHAIN_ID_PATH}/Build")
+  set(repository_hash "${HUNTER_${HUNTER_PACKAGE_NAME}_REPOSITORY_HASH}")
+  hunter_test_string_not_empty("${repository_hash}")
   set(
       HUNTER_PACKAGE_HOME_DIR
-      "${HUNTER_PACKAGE_HOME_DIR}/${HUNTER_PACKAGE_NAME}"
+      "${HUNTER_PACKAGE_HOME_DIR}/${HUNTER_PACKAGE_NAME}/${repository_hash}"
```

Comment:
As far as I understand, hunter resolve this problem by using a different root directory for each hunter release?
To emulate that here would mean setting a different hunter root directory for each possible set of (hunter release, repo A release, repo B release...). That would become quite cumbersome since packages would not be reused from one project to the next (unless the repos are exactly the same).

The other option it to read through the graph of dependencies. But this graph is not explicitly available, is it?

Did I understand this right?